### PR TITLE
changes due to making project & library public

### PR DESCRIPTION
**Pull request by meck93**
- sealer & voting-authority backend -> need bignumber.js as dev dependency due to web3js bug: https://github.com/ethereum/web3.js/issues/3386
- published @meck93/evote-crypto on npm 
- remove all .npmrc files (no longer needed)
- remove github.json and reading github config from most build scripts since it's no longer needed

### CODE REVIEW

File: voting-authority/frontend/package.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    "@typescript-eslint/parser": "^2.8.0",
-    "eslint-config-prettier": "^6.7.0",
+    "@types/react-router-dom": "^5.1.3",
+    "@types/styled-components": "^4.4.3",
+    "@typescript-eslint/eslint-plugin": "^2.27.0",
+    "@typescript-eslint/parser": "^2.27.0",
+    "eslint-config-prettier": "^6.10.1",
     "eslint-config-react": "^1.1.7",
-    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-prettier": "^3.1.2",
```
Comment (meck93) [inline voting-authority/frontend/package.json:29]: this was more of an accident but I don't think it should affect a lot...

**Comment by meck93** [comment]
I have tested the following: 
- [X] `npm install && npm run build` in a local subfolder
- [X] build `sealer` and `voting-authority` docker containers
- [X] using `./docker-prebuilt-up.sh`

**Review by alexscheitlin — APPROVED** [review]
Nice, this works as expected. Thanks!

I did the following (checks):
- install docker on a new machine
- clone the repo and this branch and create `github.json` according to the readme
- run `./docker-prebuilt-up.sh`
- play through the whole voting process
- run `npm install` in both the FE and BE of the voting-auth

**Review by nikzaugg — APPROVED** [review]
I updated the README

**Pull request merged**